### ISSUE TITLE
Issue with tooltip visible

### ISSUE THREAD

**Issue by sujalBandhara** [syncfusion/flutter-examples#48]
Hello I have found one issue when we used SplineSeries and give  animationDuration:0 then tooltip not display even it's  shouldAlwaysShow: true and enable: true.

Notes:- It's display when we give animationDuration is grater then 0 or not used that parameter.

**Comment by sujalBandhara** [comment]
Sample code for generate the issue.Issue when animationDuraton is 0 in SplineSeries.

 SfCartesianChart(                       
                // Enable legend
                legend: Legend(isVisible: false),

                // Enable tooltip
                tooltipBehavior: TooltipBehavior
                    textStyle: ChartTextStyle(color: colorThemeWhite, fontSize: 12, fontWeight: FontWeight.bold),
                    shadowColor: Colors.transparent,
                    enable: true,
                    color: colorThemePrimary,
                    canShowMarker: false,
                    format: '  point.y \t',
                    **shouldAlwaysShow: true,**
                    header: ""),
                series: <ChartSeries<UsageGraphData, String>>[
                  SplineSeries<UsageGraphData, String>(
                      splineType: SplineType.natural,                   
                      width: 3,
                      **animationDuration: 0,**
                      dataSource: chartMonthData,
                      pointColorMapper: (UsageGraphData sales, _) =>
                          sales.color,
                      xValueMapper: (UsageGraphData sales, _) => sales.year,
                      yValueMapper: (UsageGraphData sales, _) => sales.usage,
                      // Enable data label
                      dataLabelSettings: DataLabelSettings(isVisible: false))
                ])

**Comment by dharanidharandharmasivam** [comment]
Hi Sujal,

Thanks for your information. We have tried to replicate the reported issue with the provided code snippet, but to the surprise, the tooltip is showing properly. For your convenience, we have attached the [sample ](https://www.syncfusion.com/downloads/support/directtrac/general/ze/data_label-578567890)used for testing. 

Since the issue is not replicating at our end, and we suspect that the widget within which you have rendered our chart widget may cause the problem, so kindly share the entire code snippet with your data source else try to replicate the scenario with the attached sample. If possible, share the video for the issue. This will help us in investigating the issue and provide you the solution sooner.

Thanks,
Dharani.
